.travis.yml

```diff
@@ -29,11 +35,24 @@ script:
   # Build maestro
   - ./build.sh
 
+  # Run devicedb in docker
+  - |
+    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
```

Comment:
```suggestion
    if [[ "${TRAVIS_OS_NAME}" == "linux" ]]; then
```

Not manditory, but a good practice to get into. The braces can remove ambiguity when a variable is used inline with a string.

Reply:
Will fix.